config/settings/development.yml

```diff
@@ -2,6 +2,9 @@ host_inventory_url: http://inventory-web:8081
 remediations_url: https://ci.cloud.redhat.com
 rbac_url: https://ci.cloud.redhat.com
 # disable_rbac: true
+# platform_basic_auth:
+#   - username
+#   - password
```

Comment:
To test this, keep RBAC enabled, and add your basic auth credentials to `config/settings/development.local.yml`.